index.html

```diff
@@ -23,6 +23,13 @@
 </head>
 
 <body class="app">
+	<div class="browser-alarm" style="display:none;" data-bind="visible: isBrowserWarningShown()">
```

Comment:
why additional inline style here?

Reply:
at this time knockout-bindings don't work yet